src/Files.App/MainWindow.xaml.cs

```diff
@@ -3,53 +3,46 @@
 
 using Microsoft.UI;
 using Microsoft.UI.Windowing;
+using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Media.Animation;
 using Microsoft.UI.Xaml.Navigation;
 using Windows.ApplicationModel.Activation;
 using Windows.Storage;
-using WinUIEx;
 using IO = System.IO;
 
 namespace Files.App
 {
-	public sealed partial class MainWindow : WindowEx
+	public sealed partial class MainWindow : Window
 	{
 		private static MainWindow? _Instance;
 		public static MainWindow Instance => _Instance ??= new();
 
-		public IntPtr WindowHandle { get; }
+		public nint WindowHandle { get; }
 
-		private MainWindow()
+		public MainWindow()
 		{
-			WindowHandle = this.GetWindowHandle();
-
 			InitializeComponent();
 
-			EnsureEarlyWindow();
-		}
-
-		private void EnsureEarlyWindow()
-		{
-			// Set PersistenceId
-			PersistenceId = "FilesMainWindow";
```

Comment:
PersistenceId is required to separate the main window size from the properties window.

Reply:
let me take a look further 👍

Reply:
I found interesting thing; this persistence id is store in application data settings which we marks as obsolete in Files (SettingsViewModel.Get and Set)
Do we want to use this or use our settings json?

Reply:
PersistenceId is just a string which I believe AppWindow handled internally, it shouldn't have anything to do with `SettingsViewModel`.

Reply:
It uses ApplicationData.Current?.LocalSettings.Container.
That's what we had been using in SettingsViewModel.

Reply:
> It uses ApplicationData.Current?.LocalSettings.Container.

This makes sense, but it's all handled internally. It doesn't have anything to do with SettingsViewModel.

Reply:
I know but I wanted to point out this is outdated technology in Files and ask you if we want to use GeneralSettingsViewModel. This property is only in WinUIEx and we have to implement ourselves.

Reply:
What we have to implement is this property and min window size (already done).

Reply:
> I know but I wanted to point out this is outdated technology in Files and ask you if we want to use GeneralSettingsViewModel.

We shouldn't be saving or reading any settings ourselves, that is assuming the property is available in WinAppSdk (I'd be surprised if it was removed). I'm not certain if it's the same as what we had in [UWP](https://learn.microsoft.com/en-us/uwp/api/windows.ui.windowmanagement.appwindow.persistedstateid?view=winrt-26100), but there is an ID property for [WinAppSdk](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.windowing.appwindow.id?view=windows-app-sdk-1.5).

Reply:
I think we don't need it anymore.
`https://github.com/dotMorten/WinUIEx/issues/178#event-13751222808`